### PR TITLE
tomiwa profile card

### PR DESCRIPTION
**Pull request by Tomiwathedaddy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                <a href="https://twitter.com/tomiwathedaddy"><i class="fa-brands fa-twitter"></i></a>
+                <a href="https://instagram.com/tomii.wa"><i class="fa-brands fa-instagram"></i></a>
+                <a href="https://linkedin.com/in/omotoso-olatomiwa-a29359238"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="https://github.com/tomiwathedaddy"><i class="fa-brands fa-github"></i></a>
+                <a href="https://snapchat.com/tomiwathedaddy"><i class="fa-brands fa-snapchat"></i></a>
+                <a href="tommyomotouch@gmail.com"><i class="fa-solid fa-envelope"></i></a>
+            </div>
+        </div>
+        <div class="right-card">
+            <img src="./images/Profile Photo.JPG" alt="Photo">
```
Comment (Jeremiahiro) [inline index.html]: I can't find the images
Screenshot 2023-02-04 at 21 21 13

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <h2>Web Developer</h2>
+            <p>I develop websites using Bootstrap and JavaScript. Currently playing around design with Figma.</p>
+            <button class="right-btn">View Profile</button>
+            <div class="icons">
+                <a href="https://twitter.com/tomiwathedaddy"><i class="fa-brands fa-twitter"></i></a>
+                <a href="https://instagram.com/tomii.wa"><i class="fa-brands fa-instagram"></i></a>
+                <a href="https://linkedin.com/in/omotoso-olatomiwa-a29359238"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="https://github.com/tomiwathedaddy"><i class="fa-brands fa-github"></i></a>
+                <a href="https://snapchat.com/tomiwathedaddy"><i class="fa-brands fa-snapchat"></i></a>
+                <a href="tommyomotouch@gmail.com"><i class="fa-solid fa-envelope"></i></a>
```
Comment (Jeremiahiro) [inline index.html]: The email is not properly structured in the href
See what it should look like
`<a href = "mailto: abc@example.com">Send Email</a>
`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
+body{
+    background-color: #9EA9FFFF;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.card-container{
+    display: flex;
+    grid-template-columns: repeat(2, 1fr);
```
Comment (Jeremiahiro) [inline style.css]: flex and grid template columns do not work hand in hand.

**Review by Jeremiahiro — CHANGES_REQUESTED** [review]
Too many width and margin, but its fine at this stage.
Your work should be inside a folder that has your name, else we would have multiple index and style. 
Recall that you are not working alone

File: index.html (hunk 2 above)
Comment (Tomiwathedaddy) [reply index.html]: fix2

File: style.css (hunk 3 above)
Comment (Tomiwathedaddy) [reply style.css]: 'grid-template-columns' removed.

**Comment by Jeremiahiro** [comment]
Nice, this is coming into place.
Your code is missing a closing div, please review

**Pull request merged**